packages/frontend-2/components/viewer/Controls.vue

```diff
@@ -441,4 +442,12 @@ onMounted(() => {
 watch(isSmallerOrEqualSm, (newVal) => {
   activeControl.value = newVal ? 'none' : 'models'
 })
+
+onKeyStroke('Escape', () => {
```

Comment:
Preferably we want to keep all of these key stroke composables together with the rest of the code that sets up the relevant integration. There's all of those composables in postSetup that set up various integrations, I bet there's one for measurements and you can put all of these onKeyStroke invocations there with the rest of the setup.

Reply:
I have moved all measurement logic to postSetup, but this onKeyStroke is still needed here to close the activeControl (for any panel, not just measurements)